README.md

```diff
@@ -7,10 +7,65 @@ This workflow does the primary data processing for RNAseq data: alignment, QC, r
 
 ### Description
 
-The following is kind of a template protocol for a methods section. You will probably need to adapt it to your specific settings. 
+
+#### Output
+
+| File                                         | Description                                                                                                                                                                                                                                           |
+|----------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `*_merged.mdup.bam` & `.bai`                 | Standard alignment.                                                                                                                                                                                                                                   |
+| `*_chimeric_merged.mdup.bam` & `.bai`        | Chimeric alignments.                                                                                                                                                                                                                                  |
+| `_chimeric_merged.junction`                  | Correspond to the file mentioned in section “5.4 Chimeric alignments in Chimeric.out.junction” in the [STAR manual](https://github.com/alexdobin/STAR/blob/2.5.3a/doc/STARmanual.pdf). The columns and their content is described in the STAR manual. |
+| `_merged.mdup.bam.flagstat`                  | flag statistics of the “standard” bam file                                                                                                                                                                                                            |
+| `_star_logs_and_files`                       |                                                                                                                                                                                                                                                       | 
```

Comment:
no description available here

Reply:
The name says it. I don't know what is meant by "files" though.

Reply:
Can this thread be resolved?